main.go

```diff
@@ -323,10 +328,26 @@ func cmdNew(c *cli.Context) error {
 		if scanner.Err() != nil {
 			return scanner.Err()
 		}
-		file = scanner.Text()
+		title = scanner.Text()
 	}
-	file = time.Now().Format("2006-01-02-") + escape(file) + ".md"
+	file = time.Now().Format("2006-01-02-") + escape(title) + ".md"
 	file = filepath.Join(cfg.MemoDir, file)
+	t := template.Must(template.New("memo").Parse(templateMemoContent))
+	f, err := os.OpenFile(file, os.O_WRONLY|os.O_CREATE, 0644)
+	if err != nil {
+		return err
+	}
+
```

Comment:
@mattn in the next line file will be opened again (https://github.com/mattn/memo/blob/master/main.go#L330)
so using defer may make the behaviour unpredictable (trying open unclosed file stream). so i thought it's better make it explicit